components/ProductMatrix.vue

```diff
@@ -106,11 +132,15 @@ export default Vue.extend({
 
 <style lang="sass" scoped>
 .product-matrix
-  display: flex
+  display: flex // fallback
   flex-wrap: wrap
+  display: grid
+  grid-template-columns: repeat(auto-fill, minmax(142px, 1fr))
+  grid-auto-rows: minmax(142px, auto)
```

Comment:
You mean 152px?

Reply:
@coopjakob `.card` has `min-width: 142px` so I went with that

Reply:
True. I found a bug where this card was to small for the content, I will need to investigate

Reply:
There is a media query that sets it to 150px on bigger screens, so I'm going to commit a fix for that